docs/platform/tutorials/image-processing/object-detection.md

```diff
@@ -1,67 +1,122 @@
-# 3. Object detection
+# Object detection
 
-In this part of the tutorial you add an object detection service into the pipeline. This service detects objects in any video feeds connected to its input. This service uses a [YOLO v3](https://viso.ai/deep-learning/yolov3-overview/) machine learning model for object detection.
+This service takes frames from the frame grabber and detects objects in each frame. This service uses the [YOLOv8 object detection library](https://github.com/ultralytics/ultralytics){target=_blank}.
 
-In a later stage of the pipeline you add a simple UI which enables you to select the type of object to detect.
+![Object detection](../image-processing/images/object-detection-pipeline-segment.png)
 
-Follow these steps to deploy the **object detection service**:
+## 💡 Key ideas
 
-1.  Navigate to the `Code Samples` and locate `Computer Vision object detection`.
+The key ideas on this page:
 
-2.  Click `Deploy`.
+* Using the YOLOv8 library to detect objects in a frame
+* Intro to Data frame handler: `on_dataframe_received_handler`  
+* How to view logs
+* How to view the code of a Quix Application
+* Using pipeline view to examine topics
 
-3.  Click `Deploy` again.
+## What it does
```

Comment:
I'm not sure if this is the right context for what I'm about to suggest.... but generally speaking, this would be a good opportunity to talk about the resource allocation and parallel processing settings (or link to another part of the docs). 

There's some quite heavy duty processing done in this service and the reader might wonder how it all seems to happen so seamlessly (I assume it's because we've upped the memory and are using multiple replicas)

Reply:
That sounds sensible. I have a couple of tickets to write about vertical and horizontal scaling. I think it would be good to add to those from here, and talk a little about that in a scaling section (with links off to these other topics). TODO.